src/ProxyInterfaceSourceGenerator/FileGenerators/PartialInterfacesGenerator.cs

```diff
@@ -64,6 +64,10 @@ private bool TryGenerateFile(InterfaceDeclarationSyntax ci, ProxyData pd, [NotNu
         var implements = implementedInterfacesNames.Any() ? $" : {string.Join(", ", implementedInterfacesNames)}" : string.Empty;
         var @new = extendsProxyClasses.Any() ? "new " : string.Empty;
         var (namespaceStart, namespaceEnd) = NamespaceBuilder.Build(ns);
+        var events = GenerateEvents(classSymbol, proxyData.ProxyBaseClasses);
+        var properties = GenerateProperties(classSymbol, proxyData.ProxyBaseClasses);
+        var methods = GenerateMethods(classSymbol, proxyData.ProxyBaseClasses);
+        methods = methods.TrimEnd();

```

Comment:
Why only methods needs a trim?

Reply:
1 liner: `var methods = GenerateMethods(classSymbol, proxyData.ProxyBaseClasses).Trim()`;

Reply:
Sorry, I got side tracked. I started to remove some extraneous empty lines inside the output file. `methods` gets another newline inside the verbatim string. That's why i striped it.
Sorry, this should not be inside one PR.